src/ast/mod.rs

```diff
@@ -7116,6 +7123,63 @@ where
     }
 }
 
+/// Represents a list of options used in various PostgreSQL utility statements such as
+/// `EXPLAIN`, `VACUUM`, and `CLUSTER`.
+///
+/// For example, the `EXPLAIN` statement with options might look like this:
+///
+/// ```sql
+/// EXPLAIN (ANALYZE, VERBOSE true) SELECT * FROM my_table;
+/// ```
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct UtilityOptionList {
+    pub options: Vec<UtilityOption>,
+}
+
+impl Display for UtilityOptionList {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "( ")?;
+
+        let mut iter = self.options.iter().peekable();
+        while let Some(option) = iter.next() {
+            write!(f, "{}", option)?;
+            if iter.peek().is_some() {
+                write!(f, ", ")?;
+            }
+        }
+
+        write!(f, " ) ")
+    }
+}
+
+/// Represents a single PostgreSQL utility option.
+///
+/// A utility option is a key-value pair where the key is an identifier (IDENT) and the value
+/// can be one of the following:
+/// - A number with an optional sign (`+` or `-`)
+/// - A non-keyword string
+/// - keyword: `true`, `false`, `on` (`off` is also accept)
+/// - Empty
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct UtilityOption {
+    pub name: Ident,
+    pub arg: Option<String>,
```

Comment:
Should this instead be of type `Option<Expr>`? i'm thinking that would cover the described cases and be more future proof, as well as avoid the rountrip inconsistency of `FORMAT 'JSON'` => `FORMAT JSON`

Reply:
I changed it this way, though it now accepts statements beyond what the PostgreSQL parser supports.